solr/CHANGES.txt

```diff
@@ -427,6 +427,8 @@ Other Changes
 
 * SOLR-15807: New LogListener class for tests to use to make assertions about what Log messages should or should not be produced by a test (hossman)
 
+* SOLR-15897: Remove <jmx/> from all unit test solrconfig.xml files. (Eric Pugh)
+
```

Comment:
Addition here presumes backporting to `branch_9_0_0` branch, I have no views for or against that, just noting it as part of checking the entry is in the correct section :-)

Reply:
thanks!  I think I didn't quite understand how the branches worked ;-)

Reply:
Okay, I think I just did the back port...